<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Excel</class>
 <widget class="QDialog" name="Excel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Excel</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	font-size:18px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">* {
	font-size:18px;
}</string>
     </property>
     <property name="text">
      <string>请选择Excel文件:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="excel_dir"/>
     </item>
     <item>
      <widget class="QPushButton" name="choose_file">
       <property name="text">
        <string>选择Excel文件</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>选择要拆分的列:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>**无**</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>选择要输出的列:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="output_dir"/>
     </item>
     <item>
      <widget class="QPushButton" name="choose_dir">
       <property name="text">
        <string>选择输出的目录</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="do_split">
     <property name="text">
      <string>进行拆分</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
